src/lib/holocene/accordion.svelte

```diff
@@ -2,6 +2,21 @@
   import Icon from '$holocene/icon/icon.svelte';
   import Badge from '$holocene/badge.svelte';
   import type { IconName } from './icon/paths';
+  import { omit } from '$lib/utilities/omit';
+
+  type $$Props = ComponentProps<
```

Comment:
I'm not sure where the official docs live but this is implemented according to [this](https://github.com/dummdidumm/rfcs/blob/ts-typedefs-within-svelte-components/text/ts-typing-props-slots-events.md#typing-props) RFC.